BUILD.gn

```diff
@@ -68,15 +81,21 @@ if (libglfw == "bundled") {
         "Xxf86vm",
         "Xrandr",
       ]
+    } else if (target_os == "win") {
+      libs = [ "Gdi32.lib", "Shell32.lib" ]
     }
   }
 
   config("libglfw_private") {
-    cflags = [
-      "-Wall",
-      "-Wno-deprecated-declarations",
-    ]
-    defines = [ "_GLFW_USE_CONFIG_H" ]
+    if (current_toolchain != "//build/lib/win:msvc") {
+      cflags = [
+        "-Wall",
+        "-Wno-deprecated-declarations",
+      ]
+      defines = [ "_GLFW_USE_CONFIG_H" ]
```

Comment:
The cflags are toolchain-dependent, but these defines look like they would be more appropriate down in the target_os conditionals. Cross-compilation should use them too, right?

Reply:
Added new config.h instead.

Reply:
That works too 👍